layouts/shortcodes/version-docs.html

```diff
@@ -1,50 +1,62 @@
 {{- $key := .Get "key" -}}
-{{- if eq $key "PFlex_latestVersion" -}}v2.13.0
-{{- else if eq $key "PFlex_preVersion" -}}v2.12.0
-{{- else if eq $key "PStore_latestVersion" -}}v2.13.0
-{{- else if eq $key "PStore_preVersion" -}}v2.12.0
+{{- if eq $key "PFlex_latestVersion" -}}v2.14.0
```

Comment:
can you attach the local preview of the changes & screenshots related to shortcodes?

Reply:
Hi @rajkumar-palani  I have attached the screenshots. Thanks